language-support/ts/daml-types/index.ts

```diff
@@ -419,4 +420,82 @@ export const TextMap = <T>(t: Serializable<T>): Serializable<TextMap<T>> => ({
   }
 });
 
-// TODO(MH): `Map` type.
+/**
+ * The counterpart of DAML's `DA.Map.Map K V` type.
```

Comment:
Worth mentioning that this is an immutable map? And perhaps also that this is intentionally only exposed as an abstract interface and no stability of the underlying implementation is guaranteed.

Reply:
I've added a note about immutability and key equality (we use deep equality, which is different from both Immutable.js and es6 Maps). I'm not quite sure how to phrase "don't rely on things that are not exported".

Reply:
Fair point, let’s not add something about the exported stuff.